src/main/java/org/gradle/profiler/result/DurationOnlySample.java

```diff
@@ -0,0 +1,8 @@
+package org.gradle.profiler.result;
+
+public interface DurationOnlySample<T extends BuildInvocationResult> extends Sample<T> {
```

Comment:
How about `SingleInvocationSample`? Might not be worth adding a separate type for this, though.

Reply:
Done